Tools/autotest/arduplane.py

```diff
@@ -976,17 +976,16 @@ def test_throttle_failsafe_fence(self):
 
         self.load_fence("CMAC-fence.txt")
 
-        self.set_parameter("FENCE_CHANNEL", 7)
-        self.set_parameter("FENCE_ACTION", 4)
-
-        self.wait_sensor_state(mavutil.mavlink.MAV_SYS_STATUS_GEOFENCE,
-                               present=True,
-                               enabled=False,
-                               healthy=True)
+        self.set_parameter("RC7_OPTION", 11) # AC_Fence uses Aux switch functionality
+        self.set_parameter("FENCE_ACTION", 4) # Fence action Brake
         self.set_rc_from_map({
             3: 1000,
             7: 2000,
-        })
+        }) # Turn fence on with aux function
+
+
+        self.mavproxy.expect("fence enabled")
+        self.delay_sim_time(1)  # This is needed else the SYS_STATUS may not have updated
```

Comment:
if you need fo a new SYS_STATUS then just wait for it ! (self.mav.recv_match(type="SYS_STATUS", blocking=True, timeout=5)

Reply:
We're not using a `SYS_STATUS` at this point - no need to delay sim - Removed.